docs/deploy.rst

```diff
@@ -0,0 +1,228 @@
+
+.. NOTE::
+
+  This document assumes that you're `logged in`_ with your CrateDB Cloud user
+  account.
+
+.. contents::
+   :local:
+
+
+Create an organization
+======================
+
+To begin with, let's create an organization. Organizations hold projects
+that let you deploy CrateDB products such as `clusters`_. Once you've
+created an organization, you can invite other CrateDB Cloud users to become
+`organization members`_.
+
+You will automatically be the administrator of an organization that you create.
+
+Create an organization and specify the organization name, like so:
+
+.. code-block:: console
+
+    sh$ croud organizations create --name YourOrganizationName
+
+Verify that you are a member of the organization you created by giving the
+following command:
+
+.. code-block:: console
+
+    sh$ croud organizations list
+    +--------------------------------------+----------------+-------------+
+    | id                                   | name           |   plan_type |
+    |--------------------------------------+----------------+-------------|
+    | 59c7961b-33a2-4478-9e98-2a6b3f54f60e | MyOrganization |           1 |
+    +--------------------------------------+----------------+-------------+
+
+You will get a list of organizations that hold you as a member. The list shows
+the org-ID, name, and plan type of each shown organization.
+
+
+Create a project
+================
+
+Each project you create is tied to your default region. Thus also every CrateDB
+product that you deploy in a project.
+
+.. CAUTION::
+   To interact with a particular product, make sure to run the command in the
+   right region by passing the ``--region`` parameter as well as the
+   corresponding option to the command.
+
+The ``region`` setting determines in which geographical location your data is
+stored. It also specifies which CrateDB products, such as clusters, you will
+find available.
+
+You can set the following regions:
+
++-------------------+---------------------+
+| Region            | Command-line option |
++===================+=====================+
+| Azure East-US     | eastus.azure        |
++-------------------+---------------------+
+| Azure East-US     | eastus2.azure       |
++-------------------+---------------------+
+| Azure West-Europe | westeurope.azure    |
++-------------------+---------------------+
+| Bregenz           | bregenz.a1          |
++-------------------+---------------------+
+
+This tutorial uses the region ``bregenz.a1``.
+
+To set the region run the following command:
+
+  .. code-block:: console
+
+     sh$ croud config set --region bregenz.a1
+     ==> Info: Region switched to bregenz.a1
+
+Once you have set the region, all Croud commands will be executed for
+that region.
+
+Use the following command to create a project in your new organization and
+specify a project name:
+
+.. code-block:: console
+
+    sh$ croud projects create --name YourProjectName
+
+You have added a project to your organization.
```

Comment:
is there any output from this command we could include in the example instead of saying this?